web/app/components/root/root.jsx

```diff
@@ -51,28 +51,25 @@ export default class Root extends Component {
 
   componentDidMount() {
     const { sort } = this.state;
-
-    api.getUser()
-      .then(data => store.set('user', data))
-      .catch(() => store.set('user', {}))
-      .finally(() => {
-        api.find({ sort, url })
-          .then(({ comments = [] } = {}) => store.set('comments', comments))
-          .catch(() => store.set('comments', []))
-          .finally(() => {
-            this.setState({
-              isLoaded: true,
-              user: store.get('user'),
-            });
-
-            setTimeout(this.checkUrlHash);
-          });
-
-        api.getConfig().then(config => {
+    api.getConfig().then(config => {
           store.set('config', config);
           this.setState({ config });
-        })
-      });
+    });
+    Promise.all([
```

Comment:
It seems that we can do that these requests in parallel, there is no need to wait result of first to make a second